Feature/refactoring

schmunk42 opened the pull request:
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | #16  #17

schmunk42 committed:
added nginx image

schmunk42 committed:
use extension installer from mlocati/php-extension-installer, updated xdebug config

schmunk42 committed:
added docker image workflow (WIP)

schmunk42 committed:
updated workflow

schmunk42 committed:
fixed dev-tool setup

schmunk42 committed:
use master for dev-tool

schmunk42 committed:
Merge remote-tracking branch 'origin/master' into feature/refactoring

schmunk42 committed:
removed Travis

schmunk42 committed:
removed Travis badge

schmunk42 commented:
Push to Docker Hub needs to be done after merge.

samdark commented:
> Push to Docker Hub needs to be done after merge.

How?

schmunk42 commented:
> How?

The push step(s) actually are only running on `master`, we need to set the variables `DOCKER_...` (but I have no access).
But this can also done before.

rustamwin commented on README.md:
```diff
@@ -7,7 +7,7 @@
 
 **Stable**
-[![Build Status](https://travis-ci.com/yiisoft/yii-docker.svg?branch=master)](https://travis-ci.com/yiisoft/yii-docker)
+tbd
```
What does it mean?

schmunk42 replied:
to be done

when build on GH passes

samdark replied:
<img src="https://api.travis-ci.com/yiisoft/yii-docker.svg?branch=master">

samdark replied:
Could be applied now, I think.

schmunk42 replied:
added

rustamwin commented on php/Dockerfile-debian:
```diff
@@ -134,8 +60,10 @@ RUN if command -v a2enmod >/dev/null 2>&1; then \
-        -o /etc/bash_completion.d/yii
+RUN apt-get install bash-completion && \
+    mkdir /etc/bash_completion.d && \
+    curl -L https://raw.githubusercontent.com/yiisoft/yii2/master/contrib/completion/bash/yii \
```
Doesn't seem to work with yii3

schmunk42 replied:
Is there an auto-completion for yii3?
Otherwise I'd remove this.

samdark replied:
Nope, there isn't.

samdark commented:
@schmunk42 I've fixed repo permissions. Now you should have access to environment variables.

schmunk42 committed:
removed bash-completion (not available for Yii 3 at present)

schmunk42 commented:
> @schmunk42 I've fixed repo permissions. Now you should have access to environment variables.

@samdark I removed the bash completion and added the secrets.

samdark commented:
Merge time?

schmunk42 committed:
added GH-actions badge

schmunk42 commented:
Can be merged.

samdark commented:
Thank you!
